### ISSUE TITLE
The examples codesandboxes are very confusing 

### ISSUE THREAD

**Issue by 1mehdifaraji** [kidjp85/react-id-swiper#507]
Examples codesandboxes are not easy to read and figure out what does what  which should be it's main purpose . All are in typescript and have many confusing components and styles . It could be done very easily instead so it's readable and easy to understand and useable because it's very useless right now for example the swiper vertical codesandbox is a total mess and useless .

**Comment by dondrzzy** [comment]
I agree, importing this library is useless as they themselves input a lot of work to make this work in the sandbox.

**Comment by rinorzek** [comment]
agree
